src/torchjd/autojac/_utils.py

```diff
@@ -34,7 +34,7 @@ def _check_retain_graph_compatible_with_chunk_size(
 
 
 def _get_leaves_of_autograd_graph(
-    tensors: Iterable[Tensor], excluded: Iterable[Tensor]
+    tensors: Iterable[Tensor], excluded: Iterable[Tensor] | None = None
```

Comment:
I thought we typically don't have default parameters in the library core.

Reply:
True. It's convenient but it could be more work to maintain so let's revert this commit. And it's not like we have so many usages anyway.